book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] This should be within the "attr_accessor" and not the "attr_reader" thus making it something like this : ```attr_accessor :title, :author, :rentals```

Reply:
Done! Thanks